public/js/user/clinics_index.js

```diff
@@ -0,0 +1,133 @@
+const pre = "Previous";
+const next = "Next";
+const last = "Last";
+
+function getClinics() {
+  $.ajax({
+    url: route('api.clinics.index') + getQueryString(),
+    type: 'GET',
+  })
+  .done(function (response) {
+    updateNumberClinic(response.result.paginator);
+    showPaginateClinics(response.result.paginator);
+    showClinics(response.result.data);
+  });
+}
+
+function showClinics(data) {
+  data.forEach(clinic => {
+    let item =
+    `<div class="col-md-6">
+      <div class="box_list wow fadeIn">
+        <a href="#0" class="wish_bt"></a>
+        <figure>
+          <a href="detail-page.html"><img src="/images/clinic-${Math.floor(Math.random() * 5) + 1}.png" class="img-fluid" alt="" style="object-fit:cover;height:300px">
+            <div class="preview"><span>Read more</span></div>
+          </a>
+        </figure>
+        <div class="wrapper">
+          <small>Psicologist</small>
+          <h3>${clinic.name}</h3>
+
+          <p style="height:76px">${trimDescription(clinic.description)}</p>
+          <span class="rating"><i class="icon_star voted"></i><i class="icon_star voted"></i><i class="icon_star voted"></i><i class="icon_star"></i><i class="icon_star"></i> <small>(145)</small></span>
+          <a href="badges.html" data-toggle="tooltip" data-placement="top" data-original-title="Badge Level" class="badge_list_1"><img src="/images/user/badges/badge_1.svg" width="15" height="15" alt=""></a>
+        </div>
+        <ul>
+          <li><a href="#0" onclick="onHtmlClick('Doctors', 0)"><i class="icon_pin_alt"></i>View on map</a></li>
+          <li><a href="https://www.google.com/maps/dir//Assistance+–+Hôpitaux+De+Paris,+3+Avenue+Victoria,+75004+Paris,+Francia/@48.8606548,2.3348734,14z/" target="_blank"><i class="icon_pin_alt"></i>Directions</a></li>
+          <li><a href="detail-page.html">Book now</a></li>
+        </ul>
+      </div>
+    </div>`;
+    $('#js-clinic').append(item);
+  });
+}
+
+function showPaginateClinics(paginator) {
+  var prefixLink = route('user.clinics.index');
+  var paginatorClinic = $('#js-pagination-clinic');
+  var firstPage = `<li class="page-item ${disableBtnPagClinic(paginator, first)}"><a class="page-link" href="${prefixLink + '?page=1'}">${first}</a></li>`;
+  var previousPage = `<li class="page-item ${disableBtnPagClinic(paginator, pre)}"><a class="page-link" href="${prefixLink + '?page=' + getPreviousPaginatorPage()}">${pre}</a></li>`;
+  var nextPage = `<li class="page-item ${disableBtnPagClinic(paginator, next)}"><a class="page-link" href="${prefixLink + '?page=' + getNextPaginatorPage(paginator)}">${next}</a></li>`;
+  var lastPage = `<li class="page-item ${disableBtnPagClinic(paginator, last)}"><a class="page-link" href="${prefixLink + '?page=' + paginator.last_page }">${last}</a></li>`;
+  paginatorClinic.append(firstPage);
+  paginatorClinic.append(previousPage);
+  for (var i = 1; i <= paginator.last_page; i++) {
+    if (i === paginator.current_page) {
```

Comment:
Việc render HTML trong for cũng không cần thiết.
Đoạn phân trang ở ngoài view có thể luôn luôn để các button.
firstPage - previousPage - [Page 1] - nextPage - lastPage
JS thực hiện thay đổi class, URL.
Về page thì page 1 luôn luôn có, muốn có page tiếp theo thì duplicate HTML đoạn page này ra thêm/thay đổi class, URL
[Page 1] trên view không cần để class `disabled active` vào for trong JS rồi xử lý cũng được.